apps/nextjs-website/src/components/organisms/Ecosystem/Ecosystem.tsx

```diff
@@ -15,7 +15,7 @@ const Ecosystem = ({
   solutionsTabName,
   solutions,
   solutionsCta,
-}: HomepageProps['ecosystem']) => {
+}: Required<HomepageProps>['ecosystem']) => {
```

Comment:
We use `Required` here because the ecosystem attribute in `HomepageProps` is optional. However, this component only renders if the `ecosystem` attribute is present. By using `Required<HomepageProps>`, we enforce that `ecosystem` is treated as mandatory within this component. This ensures TypeScript derives the correct type, letting us work with ecosystem without additional null or undefined checks.